### PR TITLE
fix covered select options

### PR DESCRIPTION
**Pull request by B2o5T**
there is no way without using react portal

more info https://github.com/tailwindlabs/headlessui/issues/59

### CODE REVIEW

**Comment by shuding** [comment]
I think that popup is right-aligned currently, did we run into an issue with that?

**Comment by B2o5T** [comment]
@shuding it’s right aligned but in the latest PR with hided sidebar without this PR popup (switch language or theme) will be covered since container width is too small

**Comment by shuding** [comment]
Ah that makes sense!

**Review by shuding — COMMENTED** [review]
Love the new style! Can you make sure the first item is always expanded?

CleanShot 2022-09-05 at 20 06 03@2x

**Comment by B2o5T** [comment]
> Love the new style! Can you make sure the first item is always expanded?
> 
> CleanShot 2022-09-05 at 20 06 03@2x

Do we really need it? Because on mobile this will be extra expanded, more info in my comment 

https://github.com/shuding/nextra/pull/798#discussion_r961907290

If you still prefer extra expanded I’ll change it

**Comment by shuding** [comment]
Got it! I think then we only make it extend on desktop?

File: packages/nextra-theme-docs/src/index.tsx [hunk 1]
```diff
@@ -188,11 +177,21 @@ const InnerLayout = ({
...
+
   return (
-    <div
-      dir={isRTL ? 'rtl' : 'ltr'}
-      className={cn('nextra-container transition-opacity', menu && 'opacity-80')}
-    >
+    <>
+      <script
+        dangerouslySetInnerHTML={{
+          __html: `document.documentElement.setAttribute('dir','${direction}')`
```
Comment (shuding) [inline packages/nextra-theme-docs/src/index.tsx:192]: Makes sense! Does it work when the language changes like from LTR to RTL?

Comment (B2o5T) [reply packages/nextra-theme-docs/src/index.tsx:192]: yeah it works, also there is no shrinking on initial loading

**Comment by B2o5T** [comment]
will convert it to draft until I apply your suggestion

**Pull request merged**